.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,23 @@
+{
+   "configurations": [
+      {
+         "name": "Win32",
+         "includePath": [
+            "${workspaceFolder}/**",
+            "${QTDIR}/include/**"
+         ],
+         "defines": [
+            "_DEBUG",
+            "UNICODE",
+            "_UNICODE"
+         ],
+         "windowsSdkVersion": "10.0.22621.0",
+         "compilerPath": "cl.exe",
+         "cStandard": "c17",
+         "cppStandard": "c++17",
+         "intelliSenseMode": "msvc-x64",
+         "configurationProvider": "ms-vscode.cmake-tools"
+      }
+   ],
+   "version": 4
+}
```

Comment:
Some files are missing newlines

Reply:
does vscode not obey "insert_final_newline" for .editorconfig?

Reply:
We have this:
```
[*.json]
insert_final_newline = unset
```
Are the missing newlines an issue, though?

Reply:
> Are the missing newlines an issue, though?

It's not that important, it's just the way it's done historically. And without a newline, some command line tools can misbehave. For example if you `cat file` the prompt will be printed on the same line as the last line from the file

<img width="624" alt="image" src="https://github.com/user-attachments/assets/9d6c19a2-2fbe-41cf-8374-07c2e77d6b18">